source/common/network/utility.cc

```diff
@@ -342,5 +342,30 @@ bool Utility::portInRangeList(const Address::Instance& address, const std::list<
   return false;
 }
 
+absl::uint128 Utility::Ip6ntohl(const absl::uint128& address) { return flipOrder(address); }
```

Comment:
I don't think it makes sense that both of these functions do the same thing unless you actually check for local endianness somewhere. I would either do that, or use a single function, assert endianness somehow, and add a TODO.

Reply:
This has been updated to have an assert on little endianess and a todo has been added to support big endian.